### ISSUE TITLE
Add projects layer to map

### ISSUE THREAD

**Issue by TylerMatteo** [NYCPlanning/ae-cp-map#8]
# Description
Show capital projects on the map with hover functionality

## Acceptance Criteria

- [x] When the user visits the site, they see all capital projects rendered on the map
- [x] Styling for capital project points and polygons matches [specs in Figma](https://www.figma.com/design/LYHHoPop9l0jpEivk5CFzJ/Capital-Projects-Portal?node-id=670-5575&t=bHnqp3D5d54Ghn2t-0)
- [x] Styling of projects updates on hover as per Figma specs

## Out of scope
* Filtering projects shown on map by geography
* Project styling changes when selected. On-click functionality to be implemented in future Issue.

## Related work
Blocked by https://github.com/NYCPlanning/ae-zoning-api/issues/253

**Comment by TangoYankee** [comment]
@pratishta or whoever works on this, I believe we'll want to use a custom hook to hold the MVT layer

**Comment by TangoYankee** [comment]
Also, I don't see guidance on "deselection". I'm going with the easiest to implement.

1) Clicking outside the layer has no effect (because the layer doesn't know about the click. If we do want clicking outside to have an effect, then we should call this a use case for global client state and arguably a reducer)
2) Clicking on a shape that was not previously selected will select that shape and deselect any previously selected shape
3) Clicking on the a shape that was already selected will de-select it.
